### PR TITLE
Feature: Load Achievements data Android

### PR DESCRIPTION
**Pull request by RaulSangosto**
https://github.com/Abedalkareem/games_services/issues/42#issue-998932434
Android Method to get List of Achievements data.
I ve added only the id and if they are unlocked I can add the rest of the information no problem but wanted to try it before

It is working correctly

PD: Thanks for this amazing package. I love it, I hope this helps

### CODE REVIEW

File: games_services/lib/games_services.dart [hunk 1]
```diff
@@ -98,6 +98,11 @@ class GamesServices {
     return await GamesServicesPlatform.instance.getPlayerName();
   }
 
+  /// Get player score for a specific leaderboard.
```
Comment (Abedalkareem) [inline games_services/lib/games_services.dart:101]: Can you change the comment here, it's saying get player score. should be 
/// Get the list of Android Achievements data.

**Review by Abedalkareem — COMMENTED** [review]
Thanks for the pull request!

I added a minor comment.

File: games_services/lib/games_services.dart (hunk 1 above)
Comment (RaulSangosto) [reply games_services/lib/games_services.dart]: Fixed it, Should the function return a json or a list of AchievementItemData?

Comment (Abedalkareem) [reply games_services/lib/games_services.dart]: I think as you mentioned if you can return AchievementItemData that will be better!

Comment (RaulSangosto) [reply games_services/lib/games_services.dart]: I made the changes

**Review by Abedalkareem — APPROVED** [review]

**Pull request merged**